ansible/all-in-one/configs/xsnippet-webserver-api.conf.j2

```diff
@@ -1,3 +1,24 @@
+{% if xsnippet_api_rate_limits %}
+# default limit for any request (per IP)
+limit_req_zone $binary_remote_addr zone=default:10m  rate=10r/s;
```

Comment:
@ikalnytskyi  Do these numbers look ok?

Reply:
* Who is the *authenticated* user? Is this the one with passed JWT token (i.e. with `Authorization` header)?
* I think that's a good start though for authenticated users (if I understand them correctly) I'd probably increase limit to 3-5 requests per seconds, just in case.
* You mentioned 20 requests per minute so I don't see the value here. Could you please put a formula nearby how effective rate is calculated using `zone` and `rate` values?